dnf/conf/config.py

```diff
@@ -430,6 +430,36 @@ def releasever(self, val):
             return
         self.substitutions['releasever'] = str(val)
 
+        major, minor = libdnf.conf.ConfigParser.splitReleasever(str(val))
+        self.substitutions['releasever_major'] = major
+        self.substitutions['releasever_minor'] = minor
+
+    @property
+    def releasever_major(self):
+        # :api
+        return self.substitutions.get('releasever_major')
+
+    @releasever_major.setter
+    def releasever_major(self, val):
```

Comment:
It would probably be good to document that these two setters don't update the `releasever`.

Reply:
Done